src/model/annotation-list.v1.yaml

```diff
@@ -0,0 +1,50 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Annotation list
+type: object
+properties:
+    total:
+        title: Total number of Annotations
+        type: integer
+        minimum: 0
+    items:
+        title: Annotations
+        type: array
+        items:
+            type: object
+            properties:
+                uri:
+                    description: Annotation URI
+                    type: string
+                    format: uri
+                title:
+                    description: Title of the annotated document
+                    type: string
+                target:
+                    description: Target text for the annotation
+                    type: string
+                published:
+                    description: Publication date (UTC)
+                    $ref: ../misc/date-time.v1.yaml
+                updated:
+                    title: Updated date (UTC)
+                    $ref: ../misc/date-time.v1.yaml
+                content:
+                    title: Content
+                    type: array
+                    items:
+                        oneOf:
+                          - $ref: ../blocks/list.v1.yaml
+                          - $ref: ../blocks/mathml.v1.yaml
+                          - $ref: ../blocks/paragraph.v1.yaml
+                          - $ref: ../blocks/quote.v1.yaml
+                          - $ref: ../blocks/youtube.v1.yaml
+                    minItems: 1
+            required:
+              - uri
+              - title
+              - published
+              - content
```

Comment:
I think I may need to make content optional because we may wish to displayed highlighted text without an annotation.

Reply:
Ah, forgot that was a thing. Need to revert `content.minItems` as well.

Reply:
As content is optional, is it not ok to leave the `minItems` as `1`. It is my understanding that you could omit the property entirely but if it is present then it should contain at least `1` item.

Reply:
Easier to make empty the same as not there.